docs/enterprise-guide/saml-keycloak.md

```diff
@@ -0,0 +1,77 @@
+# Setting up SAML with Keycloak
+
+Keycloak is an open source platform that can be used as a user directory to save user data while acting as the IdP for single sign-on.
+
+1. [Set up SAML in Keycloak](#working-in-the-keycloak-console) (the identity provider).
+2. [Set up SAML in Metabase](../enterprise-guide/authenticating-with-saml.html) (the service provider).
+3. Once you've configured SAML in both the Keycloak console and your Metabase Admin settings, you can [check if your SSO setup is working correctly](../administration-guide/10-single-sign-on.html#checking-if-sso-is-working-correctly).
+
+For more information, check out our guide for [authenticating with SAML](authenticating-with-saml.html).
+
+## Working in the Keycloak console
+
+1. Go to the Keycloak admin console and sign in as an administrator.
+2. Create a test user from **Manage** > **Users**. You'll need to populate the fields with an email, first name, and last name.
+3. Once you've created at least one user, navigation tabs will appear at the top of the **Users** page. Go to **Credentials** to set password for your test user.
+
+- Turn off the **Temporary** toggle.
+- Click **Set Password** to save your changes.
+
+4. Create a new SSO client from **Manage** > **Clients** > **Create**.
+
+- **Client ID**: Enter “metabase” in lowercase.
+- **Client Protocol**: Select “saml” from the dropdown.
+- Click **Save**.
+
+5. Configure the SSO client from the form that appears after saving:
+
+- **Client Signature Required**: DISABLE
+- **Valid Redirect URIs**: URL where you are hosting your Metabase instance followed by
+  a slash (/) and an asterisk (_). For example, `http://localhost:3000/_`.
+- **Base URL**: Fill this in with the value under “URL the IdP should redirect back to” from your Metabase **Admin settings** > **Authentication** > **SAML**.
```

Comment:
oops, I just noticed this numbering is way off -- working on it 🤦‍♀️

Reply:
Check out: https://github.com/metabase/metabase/pull/22930